### PR TITLE
Updating critical severity lodash version.

### PR DESCRIPTION
**Pull request by taystack**
Thanks for your project and all the work you do for the open source community. Unfortunately this project is using an outdated version of `lodash`. Please refer to the screenshot I've attached. This PR should be fairly straight-forward.

Screen Shot 2019-08-27 at 3 04 42 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "version": "3.4.0",
+  "version": "3.4.1",
```
Comment (VincentLanglet) [inline package.json]: No need to change the version, it will be updated when I will create a release.

**Comment by VincentLanglet** [comment]
@taystack Thanks.
Please just revert the package version change.

**Comment by ayouidaniel** [comment]
This can be closed in favor of: https://github.com/adriantoine/enzyme-to-json/commit/b68c527ca50b426964bd3e8f0eabe4c88072f940

**Comment by VincentLanglet** [comment]
@ayouidaniel It's published

**Pull request closed without merging**